packages/wrangler/src/versions/secrets/bulk.ts

```diff
@@ -0,0 +1,128 @@
+import path from "node:path";
+import readline from "node:readline";
+import { fetchResult } from "../../cfetch";
+import { readConfig } from "../../config";
+import { UserError } from "../../errors";
+import { getLegacyScriptName } from "../../index";
+import { logger } from "../../logger";
+import { parseJSON, readFileSync } from "../../parse";
+import { validateJSONFileSecrets } from "../../secret";
+import { printWranglerBanner } from "../../update-check";
+import { requireAuth } from "../../user";
+import { copyWorkerVersionWithNewSecrets } from "./index";
+import type {
+	CommonYargsArgv,
+	StrictYargsOptionsToInterface,
+} from "../../yargs-types";
+import type { WorkerVersion } from "./index";
+
+export function versionsSecretsPutBulkOptions(yargs: CommonYargsArgv) {
+	return yargs
+		.positional("json", {
+			describe: `The JSON file of key-value pairs to upload, in form {"key": value, ...}`,
+			type: "string",
+		})
+		.option("name", {
+			describe: "Name of the Worker",
+			type: "string",
+			requiresArg: true,
+		})
+		.option("message", {
+			describe: "Description of this deployment (optional)",
```

Comment:
Generally we don't seem to be marking `optional` options (:meta: I know) this way. Is there a particular reason we want to postfix these descriptions with `(optional)`? Lack of docs perhaps?

Reply:
No idea, I was just following how we've been doing it already for secret stuff haha